{{cookiecutter.project_name}}/docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 furo=={{cookiecutter.copyright_year}}.1.2
```

Comment:
This should not be using template.. could you fix this also? It should be furo==2022.1.2

Reply:
Thanks for your review! @staticdev , you can check [here](https://github.com/cjolowicz/cookiecutter-hypermodern-python/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/docs/requirements.txt). 
This is consistent with the current version.

Reply:
I know, I just noticed since you changed this file. But this is not intended, right @cjolowicz ?

Reply:
right, this is a version number and should not be templated

Reply:
Thanks, @staticdev, and @cjolowicz! It is fixed.